docs/browsers-support/index.md

```diff
@@ -1,56 +1,42 @@
 # Browsers Support
 
+## Principles
+
+- Use native and well supported features as much as possible
+- Use new syntaxes only if they are in an advanced stage of specification, transpile them automatically using build tools
+- Avoid APIs that are not supported by the browsers your project needs to support, we don’t want to ship polyfills
+
 ## Browserslist
 
-We use [browserslist](https://github.com/browserslist/browserslist) to configure the browsers we want to support. Although this depends on the project and client requirements, our current default settings are:
+[Browserslist](https://github.com/browserslist/browserslist) allows to define the browsers to be supported in one place. Although this depends on the project and client requirements, our current defaults are:
 
 ```
 >1% in CH
 not dead
 ```
 
-We prefer to define this in a `.browserslistrc` file at the project root.
+We prefer to define this in a dedicated `.browserslistrc` file at the project root.
 
 ## CSS compatibility
 
 We use [Autoprefixer](https://github.com/postcss/autoprefixer) to automatically add vendor-prefixed properties required by the browsers defined above. Autoprefixer automatically reads Browserslist settings.
```

Comment:
As I said in my previous comment, esbuild also handles CSS natively. I'm not entirely sure about Vite integration, but it may be that `autoprefix` is no longer necessary either.

Reply:
Moved Autoprefixer to the legacy section and consequently reduced the docs about it